README.md

```diff
@@ -1 +1,43 @@
-# cloud-ops
\ No newline at end of file
+# cloud-ops
+
+## 🚀 Terraform 
+It uses a terraform script to spin up:
+1. Configure the AWS Provider
+2. Creates VPC (3 VPC)
+3. Creates a Gateway
+4. Creates Route table
+5. Creates Subnet
+6. Associate subnet with Route table 
+7. Create security group to allow ports 443, 80 and 22<br/>
+8. Create network interface with an IP from subnet <br/>
+9. Create elastic IP addres<br/>
+10. Create Ubunbut server and install apace<br/>
+
+### Run it
+To run this, initialize with `terraform init`
+This will create the .terraform and you can run it now wih the command `terraform apply`.
+Also you can run `terraform plan` first to see what the changes will be as well as to update the state.
+
+## 🚀 Hub api deployment
+
```

Comment:
We can add a basic summary of deploying using AWS, or add this section later too